server/server.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import axios from 'axios';
+import { config } from 'dotenv';
+
+const app = express();
+const port = 3001;
+
+config();
+
+app.post('/auth', async (req, res) => {
+  try {
+    const url = `https://pfa.foreca.com/authorize/token?user=${process.env.USER}&password=${process.env.PASS}`;
+    const { data } = await axios.post(url);
+    res.status(200).send(data);
+  } catch (error) {
+    res.status(400).send(error);
```

Comment:
400 - Bad Request. Did you mean to use this one?

Reply:
I was thinking 400 because the request is what could fail in the try-catch. Would using the status code from the error response object be more appropriate? I feel like 500 isn't a very good option since the server is actually fine.